scriptworker/cot/verify.py

```diff
@@ -1114,15 +1113,9 @@ def verify_decision_command(decision_link, mach_commands=DECISION_MACH_COMMANDS)
 
 # populate_jsone_context {{{1
 async def _get_additional_action_jsone_context(parent_link, decision_link):
-    actions_path = decision_link.get_artifact_full_path('public/actions.json')
     params_path = decision_link.get_artifact_full_path('public/parameters.yml')
-    action_name = get_action_name(parent_link.task)
-    all_actions = load_json_or_yaml(actions_path, is_path=True)['actions']
-    actions_tmpl = [d for d in all_actions if d['name'] == action_name][0]
     parameters = load_json_or_yaml(params_path, is_path=True, file_type='yaml')
     jsone_context = deepcopy(parent_link.task['extra']['action']['context'])
-    jsone_context['push'] = deepcopy(actions_tmpl['task']['$let']['push'])
```

Comment:
Nice improvement! I'm glad we don't have to process to open up the action template anymore.